go.mod

```diff
@@ -0,0 +1,103 @@
+module sniffer
+
+go 1.19
+
+require (
+	github.com/kubescape/go-logger v0.0.8
+	github.com/slashben/kubescape-ebpf v0.1.1
```

Comment:
[Required]: As I mentioned before, this package must pass code review like any other code that is released as part of Kubescape.

Please make sure to move transfer this code to the Kubescape organization or to create a fork so we can review and manage it properly.

Reply:
I have removed the package of kubescape-ebpf from go.mod in the interface phase